gradle/libs.versions.toml

```diff
@@ -2,7 +2,7 @@
 format.version = "1.1"
 
 [versions]
-edc = "0.0.1-milestone-9"
+edc = "0.1.0-20230527-SNAPSHOT"
```

Comment:
we could use `0.1.0-SNAPSHOT`, because `0.1.0` is around the corner

Reply:
i would stick with nightly, just to avoid surprises